arch/posix/Kconfig

```diff
@@ -20,4 +20,34 @@ config ARCH_POSIX_RECOMMENDED_STACK_SIZE
 	  thread stack, the real stack is the native underlying pthread stack.
 	  Therefore the allocated stack can be limited to this size)
 
+config ARCH_POSIX_LIBFUZZER
+	bool "Build fuzz test target"
+	help
```

Comment:
I struggled with this a bit.  Unfortunately native_posix and nrf52_bsim don't share the same main loop code, which is unfortunate.  But I came down on the side of making it an arch feature, because really that's sort of a bug with the board code organization, and that we'd **want** this to work on both eventually.